### DISCUSSION TITLE
Does the core have access to GPIO ALT functions of unexposed pins

### DISCUSSION THREAD

**Discussion by drakejest** [earlephilhower/arduino-pico#1335] [Q&A] [answered]
Im planning on making a custom board of RP2040 and use this pico core, The Rpi pico has some unexposed GPIO:

23 - used to control the buck converter
24 - VBUS voltage divider
25 - LED on board
29 - VSYS voltage divider

Does the core able to use their alt functions? PWM, i2c, uart, and SPI no problem?

**Comment by earlephilhower** [comment, marked as answer]
Sure, they're just generic GPIOs in the RP2040 itself.  The end user needs to make sure they don't end up breaking something by using them, but it's all there.

**Reply by drakejest** [reply]
Thats great to hear ! Its only me who is going to code it, i dont have anymore space to fit the whole pico board in hence i need to strip it down. Might as well use the extra pins that are in more convenient location.

Thanks !
